src/Security/AccessDeniedHandler.php

```diff
@@ -5,7 +5,8 @@
 namespace GemeenteAmsterdam\FixxxSchuldhulp\Security;
 
 use GemeenteAmsterdam\FixxxSchuldhulp\Entity\Gebruiker;
-use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Twig\Environment as EngineInterface;
+//use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
```

Comment:
Importeren als `use Twig\Environment`, zonder typecasting naar `EngineInterface`.  De typehinting in de file aanpassen. Ook de uitgecommentarieerde import kan verwijderd worden.

Reply:
Done